src/extension_instance.ml

```diff
@@ -385,6 +385,7 @@ let update_ocaml_info t =
                         ~args:[]
                     in
                     () )
+              ; ("Ignore", fun _ -> ())
```

Comment:
I don't think this is needed because there is a default close button.

Reply:
You're correct, it's not needed. I intentionally added it because I thought it might reassure people that this warning isn't a big deal. But happy to remove it if that's the consensus.

Reply:
If you can remove, it would be nice 🙂

Reply:
Done!